test/helm-test-e2e.sh

```diff
@@ -1,28 +1,41 @@
 #!/usr/bin/env bash
 
+# Copyright 2018 The Kubernetes Authors. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
 set -o errexit
 set -o nounset
 set -o pipefail
 set -o xtrace
 
 # Install and initialize helm/tiller
-HELM_URL=https://storage.googleapis.com/kubernetes-helm
-HELM_TARBALL=helm-v2.8.2-linux-amd64.tar.gz
+readonly HELM_URL=https://storage.googleapis.com/kubernetes-helm
+readonly HELM_TARBALL=helm-v2.8.2-linux-amd64.tar.gz
 
-wget -q ${HELM_URL}/${HELM_TARBALL} -P ${WORKSPACE}
-tar xzfv ${WORKSPACE}/${HELM_TARBALL} -C ${WORKSPACE}
+wget -q "$HELM_URL/$HELM_TARBALL" -P "$WORKSPACE"
```

Comment:
should this be packaged in the docker image?

Reply:
This script is for tests of a few whitelisted charts against Kubernetes master. It is not used in regular charts CI. I did not really do anything with that other than making ShellCheck happy. This should probably be tackled in a separate PR.